### PR TITLE
Update networks.json with metis testnet

### PR DESCRIPTION
**Pull request by renorzr**
add network: Metis Testnet (599)

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -943,6 +943,21 @@
...
+    "multicall": "0x5A958c159F4AEA990b1195C44624B94dC0825470",
+    "rpc": [
+      "https://goerli.gateway.metisdevops.link"
+    ],
+    "explorer": {
+      "url": "https://goerli.explorer.metisdevops.link/"
+    },
+    "start": 1018715,
+    "logo": "ipfs://QmYeskHqrEvWHqeAuqett64LxfH52HUXZi2T9BAMmgKvBF"
+  },
```
Comment (ChaituVR) [inline src/networks.json:960]: Hi, @renorzr Need to verify and publish the contract code here :)

https://goerli.explorer.metisdevops.link/address/0x5A958c159F4AEA990b1195C44624B94dC0825470/contracts

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Everything else looks good

File: src/networks.json (hunk 1 above)
Comment (renorzr) [reply src/networks.json:960]: done

File: src/networks.json [hunk 2]
```diff
@@ -959,6 +959,21 @@
     "start": 1335512,
     "logo": "ipfs://QmY1ZYBUzb46Mto7G1GitQWfaqq6n8Q4WArxFBzhNdLqvg"
   },
+  "599":{
+    "key": "599",
+    "name": "Metis Testnet",
+    "chainId": 599,
+    "network": "testnet",
+    "multicall": "0x5A958c159F4AEA990b1195C44624B94dC0825470",
```
Comment (ChaituVR) [inline src/networks.json:967]:
```suggestion
    "network": "testnet",
    "testnet": true,
    "multicall": "0x5A958c159F4AEA990b1195C44624B94dC0825470",
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**